griptape/tools/rest_api_client/tool.py

```diff
@@ -34,28 +50,27 @@ def schema_template_args(self) -> dict:
         config={
             "description": dedent(
                 """
-                This tool can be used to make a put request to the rest api url: "{{base_url}}/{{path}}".
+                This tool can be used to make a put request to the rest api url: "{{base_url}}{% if path %}/{{path}}.{% endif %}"
                 This rest api does the following: "{{description}}".
```

Comment:
@vasinov should the period go inside the quotes here? And in general, is it a good idea to quote things like this?